### PR TITLE
Add chat.tandem.Client

### PR DESCRIPTION
**Pull request by Newbytee**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** Not sure what you want me to link to here. I sent them an email and tweeted at them but got no response.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

I don't know if the icon I'm using here is redistributable. It's copied from the snap. I have no reason to believe it is redistributable, but I'm not sure how to handle it otherwise since an icon is necessary for Flathub from what I understand. How do other proprietary apps handle this?

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build chat.tandem.Client

**Comment by Newbytee** [comment]
> Build [45916 failed](https://flathub.org/builds/#/builders/32/builds/45916)

What is this gzipped appdata XML it's expecting?

File: chat.tandem.Client.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        "rm -r squashfs-root"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "chat.tandem.Client.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "chat.tandem.Client.appdata.xml"
```
Comment (barthalion) [inline chat.tandem.Client.json:82]: This file is referenced but not installed in `build-commands`.

**Comment by Newbytee** [comment]
bot, build chat.tandem.Client

**Comment by Newbytee** [comment]
bot, build chat.tandem.Client

File: chat.tandem.Client.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                "install -Dm644 chat.tandem.Client.appdata.xml /app/share/appdata/chat.tandem.Client.appdata.xml",
+                "install -Dm644 chat.tandem.Client.png /app/share/icons/hicolor/512x512/apps/chat.tandem.Client.png",
+                "install -Dm644 chat.tandem.Client.desktop /app/share/applications/chat.tandem.Client.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "unsquashfs -no-progress tandem.snap",
```
Comment (barthalion) [inline chat.tandem.Client.json:70]: I'd silence stdout here.

File: chat.tandem.Client.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <launchable type="desktop-id">chat.tandem.Client.desktop</launchable>
+  <name>Tandem</name>
+  <developer_name>Tandem Communications Inc.</developer_name>
+  <summary>Virtual office for remote teams</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://tandem.chat</url>
+  <url type="help">https://intercom.help/tandem_help</url>
+  <description>
+    <p>Connect effortlessly with your distributed team. See who’s around, talk, and collaborate in one click.</p>
```
Comment (barthalion) [inline chat.tandem.Client.appdata.xml:14]: Would be nice to have the description expanded. The marketing blurb is slightly longer on the website.

File: chat.tandem.Client.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  </description>
```
Comment (barthalion) [inline chat.tandem.Client.appdata.xml:15]: You should add a note similar to https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Ultimate/blob/master/com.jetbrains.IntelliJ-IDEA-Ultimate.metainfo.xml#L21

File: chat.tandem.Client.appdata.xml (hunk 3 above)
Comment (Newbytee) [reply chat.tandem.Client.appdata.xml:14]: I added all of the marketing "blurb", but maybe this is too much? `appstream-util` certainly thinks so. What would you include?

Comment (barthalion) [reply chat.tandem.Client.appdata.xml:14]: At least some, current description is not much more over `<summary>`.

Comment (Newbytee) [reply chat.tandem.Client.appdata.xml:14]: How about that?

Comment (barthalion) [reply chat.tandem.Client.appdata.xml:14]: LGTM

**Pull request closed without merging**